virttest/tests/unattended_install.py

```diff
@@ -1130,6 +1130,7 @@ def copy_images():
             from virttest import utils_test
             error_context.context("Copy image from NFS Server")
             utils_test.run_image_copy(test, params, env)
+            utils_test.update_qcow2_image_version(params)
```

Comment:
I think it does not matter about my question.
It's better to add update_qcow2_image_version() to run_image_copy(), which is a part of run_image_copy().